src/06-mocking-node-api/index.test.ts

```diff
@@ -11,11 +13,20 @@ describe('doStuffByTimeout', () => {
   });
 
   test('should set timeout with provided callback and timeout', () => {
-    // Write your test here
+    const callback = jest.fn();
+    jest.spyOn(global, 'setTimeout');
+    doStuffByTimeout(callback, 500);
+    expect(setTimeout).toHaveBeenCalledTimes(1);
+    expect(setTimeout).toHaveBeenLastCalledWith(callback, 500);
   });
 
   test('should call callback only after timeout', () => {
-    // Write your test here
+    const callback = jest.fn();
+    jest.spyOn(global, 'setTimeout');
+    doStuffByTimeout(callback, 500);
+    expect(callback).not.toBeCalled;
+    jest.runAllTimers;
```

Comment:
'only after timeout' mean that we need to emulate delay expire, exact how you do it in setInterval test.
this implementation not testing that stuff doing after delay, it testin that stuff run just after you run timers.